### ISSUE TITLE
Alt text stops working after reuploading a new image in a thread with an uploaded image

### ISSUE THREAD

**Issue by lindseymraz** [MbinOrg/mbin#1579]
**Describe the bug**
After creating a thread and uploading an image with alt text, editing the thread by replacing that image ensures there can't be any alt text.

**On which Mbin instance did you find the bug?**
kbin.melroy.org

**Which Mbin version was running on the instance?**
1.8.2

**To Reproduce**
1. Make a new thread. Make sure to upload an image using the button next to the language selection menu, and to give that image alt text.
2. Click to edit the thread. Upload a new image, again using the button next to the language selection menu, and again giving the image alt text.
3. Notice that the image was indeed replaced successfully, but that the new alt text doesn't show up. The old alt text doesn't show up either: there is just no alt text.

**Expected behavior**
I expect that when I follow steps 1 and 2, the new edited image has the alt text I tried to give it. 

**Smartphone (please complete the following information):**
 - Device: iPhone mini 13
 - OS: iOS 17.1.2
 - Browser: stock browser, Safari
 - Browser Version: 17.1
